com.ampl.ide.yaml

```
@@ -0,0 +1,46 @@
+app-id: com.ampl.ide
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: amplide.sh
+tags : 'proprietary'
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-documents
+modules:
+  - name: amplide
+    buildsystem: simple
+    build-commands:
+      - install -D amplide.sh /app/bin/amplide.sh
+      - install -Dm 755 apply_extra /app/bin/apply_extra
+      - install -Dm644 -t /app/extra/metainfo com.ampl.ide.metainfo.xml
+      - install -Dp -m 644 com.ampl.ide.desktop /app/share/applications/com.ampl.ide.desktop
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/com.ampl.ide.svg
+    sources:
+      - type: extra-data
+        filename: amplide.linux64.tgz
+        only-arches:
+          - x86_64
+        url: https://portal.ampl.com/dl/amplce/amplide.linux64.tgz
+        sha256: 168c30fcacc38aa0b8beee35b90b64f5560314b0696951a8a8d6bc2c6fbad399
+        size: 469596621
+        x-checker-data:
+         type: rotating-url
+         url: https://portal.ampl.com/dl/amplce/amplide.linux64.tgz
```

```suggestion
        x-checker-data:
            type: rotating-url
            url: https://portal.ampl.com/dl/amplce/amplide.linux64.tgz
```

at least use more than one space for yaml indentation.